prime-dotnet-webapi/Migrations/20200625182734_Credential.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class Credential : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "CredentialId",
+                table: "Enrollee",
+                nullable: true);
+
+            migrationBuilder.CreateTable(
+                name: "Credential",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    CreatedUserId = table.Column<Guid>(nullable: false),
+                    CreatedTimeStamp = table.Column<DateTimeOffset>(nullable: false),
+                    UpdatedUserId = table.Column<Guid>(nullable: false),
+                    UpdatedTimeStamp = table.Column<DateTimeOffset>(nullable: false),
+                    SchemaId = table.Column<string>(nullable: true),
```

Comment:
are these really nullable true?

Reply:
![image](https://user-images.githubusercontent.com/53796823/86969420-9ae57800-c122-11ea-9606-982dae37d59e.png)
I think its just not set as specifically required, but its not set to nullable true specifically either